rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import {readFileSync} from 'fs';
+import common from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+
+const banner = readFileSync('./banner.js', 'utf-8');
+const dependencies = process.env.DEPS === 'YES';
+
+export default {
+    banner: banner,
+    entry: './index',
+    format: 'cjs',
```

Comment:
I guess you could also add: `moduleName: 'opening_hours',` so that `--format iife` works directly.